gui/webview_server/static/add_provider_modal.css

```diff
@@ -24,7 +24,7 @@ h2 {
         -ms-flex: 1;
             flex: 1;
     width: 100%;
-    height: initial;
+    height: 380px;
```

Comment:
with initial, there are some rendering issues.
particularly, the overflow: scroll doesn't show up, because for some reason initial evaluates to a taller height.
tested and confirmed on thuy's laptop.